### DISCUSSION TITLE
build fail

### DISCUSSION THREAD

**Discussion by tanLyu** [halo-dev/halo#3698] [Q&A]
jdk 17.0.6
build 时报以下错，更改gradle版本依然报错，重启idea无效
Unable to find method ''java.lang.String org.gradle.api.tasks.SourceSet.getCompileConfigurationName()''.
![image](https://user-images.githubusercontent.com/23147787/230087983-78cba136-bd20-4a9f-992d-458272c7e972.png)

**Comment by JohnNiang** [comment]
建议点击途中蓝色的提示。

> 至少需要说明一下当前 build 的 git commit id 是多少。描述一下复现步骤。
